misc/systemd/kubo-hardened.service

```diff
@@ -51,26 +51,26 @@ CapabilityBoundingSet=CAP_NET_BIND_SERVICE
 # enable for 1-1024 port listening
 #AmbientCapabilities=CAP_NET_BIND_SERVICE 
 # enable to specify a custom path see docs/environment-variables.md for further documentations
-#Environment=IPFS_PATH=/custom/ipfs/path
+#Environment=KUBO_PATH=/custom/kubo/path
```

Comment:
This needs to be restored back to `IPFS_PATH`: 
we did not rename binary, we did not rename any env variables.